scripts/setup_conda_wb.sh

```diff
@@ -0,0 +1,103 @@
+# Gather workbench_path from workbench.sh
+source workbench.sh
+
+# check that workbench_path was retrieved
+if [ -z $workbench_path ]; then
+    printf "${RED}ERROR: Could not find workbench_path; please confirm 'workbench_path' entry in workbench.sh file${NC}\n"
+    exit 1
+fi
+
+# Check that retrieved path to Workbench application exists
+if [ ! -d $workbench_path ]; then
+    printf "${RED}ERROR: Path to Workbench application does not exist; please confirm 'workbench_path' entry in workbench.sh file${NC}\n"
+    printf "    ${workbench_path}\n"
+    exit 1
+fi
+
+# # ----------------------------------------------------------------------
+# Find conda in Workbench
+conda_path="${workbench_path}/rte/conda"
+if [ ! -f "${conda_path}/bin/conda" ]; then
+    printf "${RED}ERROR: Path to conda executable within Workbench application does not exist; please confirm 'workbench_path' entry in workbench.sh file${NC}\n"
+    printf "    ${conda_path}/bin/conda\n"
+    exit 1
+fi
+
+# ----------------------------------------------------------------------
+# Activate Workbench conda environment
+# >>> conda init >>>
+#!! Contents within this block are managed by 'conda init' !!
+__conda_setup="$(CONDA_REPORT_ERRORS=false "${conda_path}/bin/conda" shell.bash hook 2> /dev/null)"
+if [ $? -eq 0 ]; then
+   \eval "$__conda_setup"
+else
+   if [ -f "${conda_path}/etc/profile.d/conda.sh" ]; then
+       . "${conda_path}/etc/profile.d/conda.sh"
+       CONDA_CHANGEPS1=false conda activate base
+   else
+       PATH="${conda_path}/conda/bin:${PATH}"
+   fi
+fi
+unset __conda_setup
+# <<< conda init <<<
+
+# ----------------------------------------------------------------------
+# Check Python version available in Workbench (v3.x required)
+pyv=$(python -V 2>&1 | sed 's/.* \([0-9]\).\([0-9]\).*/\1\2/')
+if [ $pyv -lt 36 ]; then
+    printf "${RED}ERROR: Workbench with Python >= 3.6 enabled is required.${NC}\n"
+    printf "    Python version: `python --version`\n"
+    exit 1
+fi
+
+# ----------------------------------------------------------------------
+printf "${GREEN}Found Workbench:${NC} ${workbench_path}\n"
+printf "${GREEN}Activated Conda: ${NC} ${conda_path}\n"
+
+yes | python -m pip install watts
+
+printf "${GREEN}WATTS successfully installed in Workbench environment.\n"
+
+# -----------------------------------------------------------------------
+# Copying files between WATTS and Workbench
+printf "${GREEN}Transfering files between WATTS and Workbench.\n"
+
+script_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
+
+cd ../src/watts_ui
+
+current_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
+
+# Copy watts.py to Workbench/rte
+cp etc/watts.py $workbench_path/rte/
+
+# Check if bin directory already exists
+if [ -d "$current_path/bin" ]; then
+    printf "${RED}ERROR: bin directory already exists in $current_path.${NC}\n"
+    exit 1
+fi
+
+mkdir bin
+ln -s $workbench_path/bin/sonvalidxml bin/sonvalidxml
```

Comment:
this will crash if we already have `bin` there - which will be the case if we are updating `workbench` install.

Reply:
Allowed transfer to skip if `bin` already exists